content/docs/tutorials/getting-started-with-trust-manager/README.md

```diff
@@ -0,0 +1,600 @@
+
+> **Note:** All resources provided are demonstrative and should be reviewed 
+  properly before using in production environments.
+
+## Prerequisites
+
+**💻 Software**
+
+1. [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl): The Kubernetes
+command-line tool which allows you to configure Kubernetes clusters.
+1. [helm](https://helm.sh/): A package manager for Kubernetes.
+1. [yq](https://github.com/mikefarah/yq#install): A command line tool for
+parsing YAML with helpful coloring.
+
+## Distribute Public CA Trust
+
+Let us first setup trust-manager and have our public CAs distributed to our
+demo namespace: `team-a`.
+
+### Setup Application & Bundle
+
+1) Ensure you have [trust-manager](../../projects/trust-manager/README.md#installation) installed. If not simply use:
+
+    ```shell
+    helm repo add jetstack https://charts.jetstack.io
+    helm upgrade -i -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true --wait --create-namespace
+    helm upgrade -i -n cert-manager trust-manager jetstack/trust-manager --wait
+    ```
+
+1) Create your first `Bundle` resource including only Public CA certificates
+
+    ```yaml file=./trust/bundle-public.yaml
+    ```
+
+    ```shell
+    kubectl apply -f - <<EOF
+    apiVersion: trust.cert-manager.io/v1alpha1
+    kind: Bundle
+    metadata:
+      name: public-bundle
+    spec:
+      sources:
+      - useDefaultCAs: true
+      target:
+        configMap:
+          key: "ca-certificates.crt"
+        namespaceSelector:
+          matchLabels:
+            trust: enabled
+    EOF
+    ```
+
+1) Let's create a namespace where our application will run:
+
+    ```shell
+    kubectl apply -f - <<EOF
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      labels:
+        kubernetes.io/metadata.name: team-a
+        trust: enabled
+      name: team-a
+    EOF
+    ```
+
+    Note that this namespace is labelled with `trust: enabled` which matches the
+    `namespaceSelector` criteria in the `Bundle` resource:
+
+    ```yaml
+      namespaceSelector:
+        matchLabels:
+          trust: enabled
+    ```
+
+1) Verify that the trust-manager controller has correctly propagated the
+CA bundle to the namespace:
+
+    ```shell
+    kubectl get configmap -n team-a public-bundle -o yaml
+    ```
+
+    Note that this output should be quite long. This is because the default
+    public bundle that we use has a lot of public CAs in it.
+
+### Mount Trust Bundle to Application with Automatic Use
+
+Now we will mount our trusted CAs to the application in the default location
+that most applications expect to find a `ca-certificates.crt` file. The benefit
+to this approach is that any application code inside the container will use
+this file by default and not require any additional configuration. There is
+the added benefit that you will be mounting over the top of 
+`/etc/ssl/certs` which will remove any existing CA certificates usually
+present from a container base image.
```

Comment:
suggestion: This is our first mention of _where_ the cert bundle should be mounted - we talk about the nuance of this later (i.e. that it differs depending on the OS your base image was based on) but is there a risk that users miss the later bit and they end up trying to mount `/etc/ssl/certs` on, say, a RHEL image and get confused?

Would it be worth adding a tiny "disclaimer" here saying that the location can vary?

Reply:
I have added a `WARNING` section, can you review and see if that helps / provides sufficient warning to clients.

A small plug for paranoia too as it might be useful to people following this.